packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -942,7 +942,8 @@ describe("Runtime", () => {
 				);
 
 				// The entryPoint should be undefined because the deprecated load() method was used
-				const actualEntryPoint = await containerRuntime.getEntryPoint();
+				assert(containerRuntime.getEntryPoint !== undefined); // The function should exist, though
+				const actualEntryPoint = await containerRuntime.getEntryPoint?.();
```

Comment:
I think the `?` is optional here if you assert about it above.

Reply:
Ah, true, I mixed this with _our_ assert which doesn't have a type guard. I'll merge like this for expediency.